ReactWindows/ReactNative/Accessibility/DynamicAutomationPeer.cs

```diff
@@ -45,7 +48,7 @@ protected override object GetPatternCore(PatternInterface patternInterface)
             {
                 return this;
             }
-            return null;
+            return base.GetPatternCore(patternInterface);
```

Comment:
I didn't notice a difference in the trees. I did this because ListVIewItemAutomationPeer defers pattern to base class. I can put a check for this to be done only if trait is ListItem though

---
In reply to: [184141271](https://github.com/Microsoft/react-native-windows/pull/1803#discussion_r184141271) [](ancestors = 184141271)

Reply:
I see. If it's for listitem control type, I understand.

---
In reply to: [184221781](https://github.com/Microsoft/react-native-windows/pull/1803#discussion_r184221781) [](ancestors = 184221781,184141271)